src/lib/stable/stable_structures/stable_b_tree_map.ts

```diff
@@ -5,13 +5,23 @@ export interface Serializable {
     fromBytes: (bytes: Uint8Array) => any;
 }
 
-// TODO make this function's return type explicit https://github.com/demergent-labs/azle/issues/1860
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export type StableBTreeMap<Key, Value> = {
```

Comment:
Let's see if we cannot duplicate these types somehow, try index types on the StableBTreeMap

Reply:
it was not readily apparent what to do, we decided it would be better to repeat ourselves than have the TODO